superset-frontend/src/dashboard/components/FiltersBadge/DetailsPanel.tsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { useTheme } from '@superset-ui/core';
+import {
+  SearchOutlined,
+  MinusCircleFilled,
+  CheckCircleFilled,
+  ExclamationCircleFilled,
+} from '@ant-design/icons';
```

Comment:
Hmm, it may be better to find a way to pull these from `/src/common/components` - but then it seems we actually now have _two_ `Icon` components that will be competing with each other... one for AntD stuff, one for custom icons from Figma. Hopefully we can find a way to consolidate, as there's a lot of overlap. Perhaps not a concern for this PR, but it raised my eyebrow